tools/generate-changelog.js

```diff
@@ -0,0 +1,135 @@
+const { execSync } = require('child_process')
+const fs = require('fs').promises
+
+const versionRegex = /([\d.\d.\d]+)/
+const taskNumberRegex = /(FOROME-\d+)/
+const delimeter = '??'
+const maxBuffer = 50 * 1024 * 1024
+const logFormat = `--pretty=format:'%h${delimeter}%an${delimeter}%ad${delimeter}%s${delimeter}%d'`
+const techCommitMessage = 'docs:changelog-auto-generated'
+const techAccountName = 'Version Bot'
+const baseJiraUrl = 'https://quantori.atlassian.net/browse/FOROME-'
+const baseGitHubUrl =
+  'https://github.com/ForomePlatform/Anfisa-React-Client/commit/'
+
+const execCommand = (command, options) => {
+  return new Promise((resolve, reject) => {
+    let output
+    try {
+      output = execSync(`npx ${command}`, options)
```

Comment:
you can use `exec` instead of `execSync` if you want promise

Reply:
> you can use exec instead of execSync if you want promise
exec doesn't return a promise so I'll need to promisify it anyway
>Also, why you use npx for git command
Thanks, forgot to remove